src/Raven.Server/Routing/RouteInformation.cs

```diff
@@ -111,10 +113,26 @@ public Task CreateDatabase(RequestHandlerContext context)
                         if (storedEtag < etag)
                             hasChanges = true;
                     }
-                    else
+                    else if (etag > 0)
                     {
-                        if (etag > 0)
+                        hasChanges = true;
+                    }
+
+                    if(hasChanges == false)
+                    {
+                        DatabaseTopology topology = null;
+                        
+                        if (context.HttpContext.Request.Query.TryGetValue("nodetag", out var nodeTag))
```

Comment:
hmm after more though about it, 
We can read cluster topology + the host name from the request url and know which node is trying to connect

Reply:
so we don't need any of the above

Reply:
not relevant because the hostname from the request is not the hostname of the sender